rbx_reflector/src/cli/mod.rs

```diff
@@ -29,7 +29,7 @@ impl Args {
     pub fn run(&self) -> anyhow::Result<()> {
         match &self.subcommand {
             Subcommand::Dump(sub) => sub.run(),
-            Subcommand::DefaultsPlace(sub) => sub.run(),
+            Subcommand::DefaultsPlace(sub) => sub.run().map(|_| ()),
```

Comment:
...What is this?

Reply:
The defaults place subcommand now returns Result<StudioInfo> instead of Result<()>, and this top-level run function still returns a Result<()>. I wanted to continue to propagate any error information, so chose to just map back to a unit